### ISSUE TITLE
[Feature Request] Sync Primary and Clipboard buffers

### ISSUE THREAD

**Issue by abyss7** [Tudmotu/gnome-shell-extension-clipboard-indicator#227]
It's a main feature that I use with other clipboard managers: select with mouse, paste with <kbd>Ctrl</kbd>+<kbd>C</kbd>. Unfortunately, it's not available with this extension.

**Comment by salim-b** [comment]
I don't think this is a good idea to mix them. There are three different selections (PRIMARY, SECONDARY, CLIPBOARD) for a reason. The usual convention is to **paste PRIMARY** (what's currently selected with mouse/keyboard) **using the middle mouse button**, see the [Freedesktop.org Specification](https://specifications.freedesktop.org/clipboards-spec/clipboards-latest.txt).

**Comment by LordChaos73** [comment]
Aside how content gets pasted, if I select something it doesn't end up in the clipboard's history which is a feature I dearly miss.
